src/utils/upload.js

```diff
@@ -0,0 +1,76 @@
+const crypto = require('crypto');
+const request = require('request-promise');
+const retry = require('bluebird-retry');
+
+const digestHash = crypto.createHash('md5');
```

Comment:
The Hash object can not be used again after hash.digest() method has been called. Multiple calls will cause an error to be thrown.